### DISCUSSION TITLE
Roll at level

### DISCUSSION THREAD

**Discussion by axelisart** [iNavFlight/inav#6390] [Q&A]
Hi, my quad is displaying a roll of around -2 degrees when placed at level (setup tab in Inav configurator). If I boot up the FC (Kakute F7 v1.5) and I don't touch anything, it always display some roll after autocalibration at startup. I don't understand why and I suspect it to be linked to my problem of quad drifting in stabilized mode. To me just after booting the FC, the roll and pitch should be at zero (which is the case of pitch in my case but not roll) Can anybody help ?
Thanks.

**Comment by DzikuVx** [comment]
Please do remmeber, that gyro/acc on those board is ona foam pad. And the error might be coming from the fact, that acc is just not in parallel to the ground at all

**Comment by axelisart** [comment]
Yes I know, but I though that the calibration (6 steps calibration) has the ability to take care about that and compensate for it. Well I changed the foam and now I just have one degree of error on roll axis. Will test it tomorow and say if it reduce my drift problem.
Thanks

**Comment by axelisart** [comment]
Well, this did not solve my problem. Maybe is it because I switch from Accro to Angle mode during flight ? I'm investigating for a long time on what could cause that but I still don't find the root cause.
